cmd/api/src/model/filter.go

```diff
@@ -344,6 +352,6 @@ func (s QueryParameterFilterParser) ParseQueryParameterFilters(request *http.Req
 
 func NewQueryParameterFilterParser() QueryParameterFilterParser {
 	return QueryParameterFilterParser{
-		re: regexp.MustCompile(`([\w]+):([\w\d\--_]+)`),
+		re: regexp.MustCompile(`([~\w]+):([\w\--_ ]+)`),
```

Comment:
I created a regexr suite of tests to validate the existing predicates and the new one
https://regexr.com/8583j

Reply:
Golang correctly parses `+` and `%20` and replaces them with spaces, the empty character in the regex string is intentional so that we can support spaces in search terms

Reply:
I'm not sure how easy it is but if it's not too much more work, I think having a unit test around this regex would be a nice addition!

Reply:
We already have a unit test for the function, but I can definitely add additional scenarios to it!
https://github.com/SpecterOps/BloodHound/blob/3a193574fad089963f433b6f2a3ba5ac2d0c41c1/cmd/api/src/model/filter_internal_test.go#L25

Reply:
Yes please! 🙇